templates/digital-asset-template/scripts/move/publish.js

```diff
@@ -4,32 +4,55 @@ const yaml = require("js-yaml");
 const cli = require("@aptos-labs/ts-sdk/dist/common/cli/index.js");
 
 const config = yaml.load(fs.readFileSync("./.aptos/config.yaml", "utf8"));
-const accountAddress =
-  config["profiles"][process.env.VITE_APP_NETWORK]["account"];
+const accountAddress = config["profiles"][process.env.VITE_APP_NETWORK]["account"];
```

Comment:
this needs to be changed, look at https://github.com/aptos-labs/create-aptos-dapp/pull/122